### PR TITLE
Sameer/refactor

### PR DESCRIPTION
**Pull request by crazzywizard**
### Purpose
* play with framer motion
* refactored code to fix buggy screen loading
* switched roadmap screen from magic numbers to enum

### CODE REVIEW

File: components/PickCre8orType/pickCre8orType.tsx [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
-            >
-              <div className="px-6 py-4">
-                <h4 className="mb-2 text-xl font-semibold tracking-tight text-gray-800">
-                  {cre8orType.title}
-                </h4>
-                <p className="leading-normal text-gray-700">{cre8orType.description}</p>
-              </div>
-            </button>
-          ))}
+      <motion.div
```
Comment (sweetmantech) [inline components/PickCre8orType/pickCre8orType.tsx:11]: these are those new animations right?

File: components/Roadmap/Roadmap.tsx [hunk 2]
```diff
@@ -1,19 +1,42 @@
...
+      id: "coming-soon1",
+    },
+    {
+      displayText: "COMING SOON",
+      id: "coming-soon2",
+    },
+    {
+      displayText: "COMING SOON",
+      id: "coming-soon3",
+    },
```
Comment (sweetmantech) [inline components/Roadmap/Roadmap.tsx:39]: what's the difference between this `RoadmapCards` & `RoadmapScreens` in `/lib/enums`?

can these be DRY'd up into one variable?

File: lib/enums.ts [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
-  TypeformQuiz = 4,
-  Details = 5,
+export enum RoadmapScreens {
+  Roadmap = "roadmap",
+  Trailer = "trailer",
+  Mysteries = "mysteries",
+  Storyline = "story",
+  AllowListChoice = "allow-list-choice",
+  ComingSoon = "roadmap",
+}
```
Comment (sweetmantech) [inline lib/enums.ts:13]: same question here, can these be combined with `RoadmapCards` in `components/Roadmap/Roadmap`?

**Review by sweetmantech — APPROVED** [review]
ship it 🚢

File: components/Roadmap/Roadmap.tsx (hunk 2 above)
Comment (crazzywizard) [reply components/Roadmap/Roadmap.tsx:39]: Roadmap Screens is akin to active step values. Roadmap cards is just array with card values so we can add additional attributes to it like image value, different class name if we want different roads to have different colors or what not. I changed the variable from array of strings to array of objects

File: components/PickCre8orType/pickCre8orType.tsx (hunk 1 above)
Comment (crazzywizard) [reply components/PickCre8orType/pickCre8orType.tsx:11]: Yep!

File: lib/enums.ts (hunk 3 above)
Comment (crazzywizard) [reply lib/enums.ts:13]: This may be possible if each card is a separate page.

**Pull request merged**